core/tests/test_asset_incident_report_model.py

```diff
@@ -23,3 +28,48 @@ def test_model_string_representation(self):
             str(incident_report),
             f"{incident_report.incident_type}: {incident_report.asset}",
         )
+
+
+class TestAssetIncidentReportNotifications(CoreBaseTestCase):
+    """
+    Test Notifications elicited by Asset Incident Report creation
```

Comment:
Just for semantics sake and nothing related to tech, `elicited` isn't the right word to use there. looool

Reply:
Noted and fixed